### PR TITLE
[next] Production Fixes + Initial Documentation

### PR DESCRIPTION
**Pull request by wyattjoh**
## What does this PR do?

- Added README, LICENSE, fixed bugs with package.json

## How do I test this PR?

Wait for CI to pass!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
+# talk
```
Comment (alinhle) [inline README.md:1]: Talk with a big T?

File: README.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+# Create the `docker-compose.yml` file to get started.
+cat > docker-compose.yml <<EOF
+version: "2"
+services:
+  talk:
+    image: coralproject/talk:next
+    restart: always
+    ports:
+      - "127.0.0.1:3000:5000"
```
Comment (alinhle) [inline README.md:40]: What's at port 5000?

File: README.md [hunk 3]
```diff
@@ -0,0 +1,141 @@
...
+npm run start
+```
+
+Then head on over to http://127.0.0.1:3000 to install Talk!
+
+## Configuration
+
+The following environment variables can be set to configure the Talk Server:
+
+- `NODE_ENV` - Can be one of `production`, `development`, `test`. All production
```
Comment (alinhle) [inline README.md]: Not sure if we should show `test` here, as this is only for local dev running tests.

File: README.md (hunk 2 above)
Comment (wyattjoh) [reply README.md:40]: Only Docker defaults to 5000, as 3000 is typically considered a development port.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,141 @@
...
+  deployments should use `production`. Defaults to `production` when ran with
+  `npm run start` and `development` when run with `npm run start:development`.
+- `PORT` - The port to listen for HTTP and Websocket requests. (Default `3000`)
+- `MONGODB_URI` - The MongoDB database URI to connect to.
+  (Default `mongodb://127.0.0.1:27017/talk`)
+- `REDIS_URI` - The Redis database URI to connect to.
+  (Default `redis://127.0.0.1:6379`)
+- `SIGNING_SECRET` - The shared secret to use to sign JSON Web Tokens (JWT) with
+  the selected signing algorithm. (Default: `keyboard cat`)
```
Comment (alinhle) [inline README.md:114]: Does this mean it'll prompt for a signing secret?

Comment (wyattjoh) [reply README.md:114]: For the moment, it defaults to `keyboard cat`, but I have a `TODO` to revisit that behavior.

File: README.md (hunk 1 above)
Comment (wyattjoh) [reply README.md:1]: I typically place the first line equal to the repository name, which in this case, is lowercased.

**Review by alinhle — APPROVED** [review]

**Pull request merged**